src/site/content/en/reliable/codelab-background-sync/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: codelab
+title: Building a resilient search experience with Workbox
+authors:
+  - demianrenzulli
+  - jeffposnick
+date: 2020-06-17
+description: |
+  How to implement a resilient search experience with Workbox, using  Background Sync and the Push API.
+glitch: resilient-search-codelab
+---
+
+This codelab shows you how to implement a resilient search experience with Workbox. The demo app it uses contains a search box that calls a server endpoint, and redirects the user to a basic HTML page.
+
+{% Aside %} 
+This codelab uses [Chrome DevTools](https://www.google.com/chrome/). Download Chrome if you don't already have it. 
+{% endAside %}
+
+## Measure
+
+Before adding optimizations, it's always a good idea to first analyze the current state of the application.
+
+{% Instruction 'remix' %}
+{% Instruction 'preview' %}
+
+Check how the website behaves when going offline:
+
+{% Instruction 'devtools-network', 'ol' %}
+1. Open Chrome DevTools and select the Network panel.
+1. In the [Throttling drop-down list](https://developers.google.com/web/tools/chrome-devtools/network/reference#throttling), select **Offline**.
+1. In the demo app enter a search query, and click on the search button.
+
+The standard browser error page is shown:
+
+![A screenshot of the default offline UX in the browser.](offline-ux-default.png)
+
+## Provide a fallback response
+
+The service worker contains the code to add the offline page to the [precache list](https://developers.google.com/web/tools/workbox/modules/workbox-precaching#explanation_of_the_precache_list), so it can always be cached at the service worker "install" event.
```

Comment:
```suggestion
The service worker contains the code to add the offline page to the [precache list](https://developers.google.com/web/tools/workbox/modules/workbox-precaching#explanation_of_the_precache_list), so it can always be cached at the service worker `install` event.
```
We usually format the literal names of JS events as code